docs/contracts.rst

```diff
@@ -1201,7 +1201,7 @@ contracts (``L.f()`` if ``L`` is the name of the library). Furthermore,
 ``internal`` functions of libraries are visible in all contracts, just as
 if the library were a base contract. Of course, calls to internal functions
 use the internal calling convention, which means that all internal types
-can be passed and memory types will be passed by reference and not copied.
+can be passed and types `stored in memory <types.html#data-location>`_ will be passed by reference and not copied.
```

Comment:
I think we should use references as part of RST. I think the syntax is something like `:ref:data-location`

Reply:
@axic Yeah, this was kind of a shortcut as the other PR mentioned above adds new links anyway, hmm, I'll redo it properly anyway and we can worry about those conflicts when it comes to merging things later.